pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_products.adoc

```diff
@@ -5,6 +5,6 @@ A _product_ in Pantheon is a category that corresponds to a product in the conve
 
 You assign the product name and version in the metadata of a module or assembly. That module or assembly then appears in the search results for that product.
 
-{ProductShortName} does not allow you to add a product name if that name already exists.
+When you create a new product, {ProductShortName} does not allow you to use a product name if a product with that name already exists. Only a Publisher can create a new product.
```

Comment:
The opening sentence here feels awkward to me, but I can't pinpoint exactly what's wrong with it. Just wanted to note that it reads strangely to me. Maybe this would be better? I'm not positive that it conveys the same information, but I think it does:

```adoc
{ProductShortName} does not allow you to create a new product with a product name that already exists.
```